src/custom/CortiAuth.ts

```diff
@@ -116,21 +135,46 @@ export class Auth extends FernAuth {
         }, requestOptions));
     }
 
+    /**
+     * Patch: PKCE-specific method for Authorization code flow
+     */
+    public getCodePkceFlowToken(
+        request: AuthorizationCodePkceServer,
+        requestOptions?: FernAuth.RequestOptions,
+    ): core.HttpResponsePromise<Corti.GetTokenResponse> {
+        return core.HttpResponsePromise.fromPromise(this.__getToken_custom({
+            ...request,
+            grantType: "authorization_code",
+        }, requestOptions));
+    }
+
     /**
      * Patch: copy of this.__getToken with patches
      */
     private async __getToken_custom(
         /**
          * Patch: added additional fields to request to support Authorization code flow
          */
-        request: Corti.AuthGetTokenRequest & Partial<{
+        request: (Corti.AuthGetTokenRequest | AuthorizationCodeServer | AuthorizationCodePkceServer | AuthorizationRefreshServer) & Partial<{
```

Comment:
We don't really need to list all the types in here, because this is private internal method, that we know how to use, and all customizations and testing are coming from the `public` methods. Also we just "strip" some of the values later, and `Partial` thing will have all these fields... So types unfortunately don't add any value in this case.

In perfect world this would be auto-generated, actually. It's just they don't support some of the staff we need, this is why we manually adding some logic here (like default grantType for example). And less changes we do here -- the easier it would be to support, when they will generate what we need